.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: RPC Tests
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
I think Isita mentioned this in the Corev4 File, but is there a reason why it's just Ubuntu?